impl_core/src/test/java/io/opencensus/implcore/trace/export/SampledSpanStoreImplTest.java

```diff
@@ -83,7 +88,7 @@ public void setUp() {
     sampleStore.registerSpanNamesForCollection(Arrays.asList(REGISTERED_SPAN_NAME));
   }
 
-  SpanImpl createSpan(String spanName) {
```

Comment:
Should we keep the name? To ensure backward compatibility?

Reply:
It is test (so no need) (nobody links this) + it is part of implementation package where we don't promise backwards compatibility.